src/vivarium_dcpn_hypertension_mgt/tools/utilities.py

```diff
@@ -79,10 +148,8 @@ def collapse_other_drug_profiles(data):
 
         collapsed_profile = category_profiles.iloc[[0]]
         collapsed_profile['drug_class'] = 'other'
-        collapsed_profile['percentage_among_therapy_category_mean'] = \
-            category_profiles.percentage_among_therapy_category_mean.sum()
-        collapsed_profile['percentage_among_therapy_category_sd'] = \
-            (category_profiles.percentage_among_therapy_category_sd ** 2).sum()
+        collapsed_profile['mean'] = category_profiles['mean'].sum()
+        collapsed_profile['sd'] = ((category_profiles.sd ** 2).sum()) ** 0.5
```

Comment:
Thank you @halfhorst for catching that I wasn't square rooting this in the last version!